cmd/wait/runner.go

```diff
@@ -199,5 +199,65 @@ func (r *runner) run(ctx context.Context, _ *cobra.Command, _ []string) error {
 		r.logger.LogCtx(ctx, "message", "CoreDNS is ready")
 	}
 
+	if r.flag.Provider != "kvm" {
```

Comment:
I think it could help future maintainers to link to an issue or add a short comment about why we only check this on KVM--not that the rest of `standup` is very well commented as it is.

Reply:
I will add a comment but this should be everything except KVM, right?

Reply:
Yeah, I mean why this is provider-specific at all.